packages/polymarket-notifier/src/polymarketNotifier.js

```diff
@@ -160,17 +159,27 @@ class PolymarketNotifier {
 
   // gets Polymarket API data that can be used to compare against proposals
   async getQuestionData() {
-    const binaryAdapterContract = await new this.web3.eth.Contract(binaryAdapterAbi, binaryAdapterAddress);
     const ctfAdapterContract = await new this.web3.eth.Contract(ctfAdapterAbi, ctfAdapterAddress);
 
     // Polymarket API
-    const apiUrl = this.apiEndpoint + "?_limit=-1&active=true&_sort=created_at:desc";
-    const polymarketContracts = await this.networker.getJson(apiUrl, { method: "get" });
+    const query = `
+    {
+      markets(where: "active = true AND question_ID IS NOT NULL and (resolved_by = '${binaryAdapterAddress}' OR resolved_by = '${ctfAdapterAddress}')", order: "created_at desc") {
```

Comment:
We need to filter the resolved_by. There are only 3 markets removed by that filter"

And they correspond to a resolved_by == gnosis safe

```
{
  "data": {
    "markets": [
      {
        "resolvedBy": "0x0dD333859cF16942dd333D7570D839b8946Ac221",
        "questionID": "0x6f3fdb3e9fb581e31200725d0496f13a430ea715400952e9f16e50654551e29e",
        "createdAt": "2022-04-08 14:43:56.612+00",
        "question": "NHL: Will the Hurricanes beat the Islanders by more than 1.5 goals in their April 8 matchup (7 PM ET)?",
        "outcomes": "[\"Yes\", \"No\"]",
        "outcomePrices": "[\"0.0000008357364822480705152700290162813914\", \"0.9999991642635177519294847299709837\"]",
        "umaResolutionStatus": null,
        "startDate": "2022-04-08T18:01:57.980Z",
        "resolutionSource": "https://www.nhl.com/scores/"
      },
      {
        "resolvedBy": "0x0dD333859cF16942dd333D7570D839b8946Ac221",
        "questionID": "0x759f3081b28634408d4ab346bbf8ec522cd7f845b6f03fad45b3cb88e1e6c399",
        "createdAt": "2022-04-08 14:40:58.569+00",
        "question": "NHL: Will the Lightning beat the Bruins by more than 1.5 goals in their April 8 matchup (7 PM ET)?",
        "outcomes": "[\"Yes\", \"No\"]",
        "outcomePrices": "[\"0.0000008329839365665547409453218649933631\", \"0.999999167016063433445259054678135\"]",
        "umaResolutionStatus": null,
        "startDate": "2022-04-08T18:00:28.338Z",
        "resolutionSource": "https://www.nhl.com/scores/"
      },
      {
        "resolvedBy": "0x0dD333859cF16942dd333D7570D839b8946Ac221",
        "questionID": "0x51b69aec0e5a66ab76cebf5f0170e0899036bf2b286593932d952fdb7d11d1c3",
        "createdAt": "2022-02-11 21:38:24.32+00",
        "question": "What will the price of Polygon ($MATIC) be on March 10, 2022? ",
        "outcomes": "[\"Long\", \"Short\"]",
        "outcomePrices": "[\"0.2107292542510757213280057655380775\", \"0.7892707457489242786719942344619225\"]",
        "umaResolutionStatus": null,
        "startDate": null,
        "resolutionSource": "https://www.coingecko.com/en/coins/polygon"
      },
      {
        "resolvedBy": "0x0dD333859cF16942dd333D7570D839b8946Ac221",
        "questionID": "0x6f206124b89b6dc960e6f9dd1d6bc31a369be1683ae2b72e62b065dc9d2e17bc",
        "createdAt": "2022-02-11 21:35:56.22+00",
        "question": "What will the price of Ethereum ($ETH) be on March 10, 2022? ",
        "outcomes": "[\"Long\", \"Short\"]",
        "outcomePrices": "[\"0.2971387852280070203612081088672809\", \"0.7028612147719929796387918911327191\"]",
        "umaResolutionStatus": null,
        "startDate": "2022-02-11T21:50:22.231Z",
        "resolutionSource": "https://www.coingecko.com/en/coins/ethereum"
      }
    ]
  }
}
```

Reply:
Makes sense!